framework/components/ecl-search-forms/ecl-search-forms.twig

```diff
@@ -1,5 +1,27 @@
+{#
+  - variant: can be '','internal' (default: '')
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-search-form' ~ (variant != '' ? ' ecl-search-form--' ~ variant: '') %}
```

Comment:
In the current implementation in the site header, the check adds the variant/modifier part unnecessary:

![screenshot from 2017 08 16 13 12 29](https://user-images.githubusercontent.com/1923476/29360809-ec533afe-8284-11e7-97a3-bea937e4bccc.png)